models/incident.js

```diff
@@ -69,20 +69,16 @@ var Incident = mongoose.model('Incident', schema);
 schema.plugin(autoIncrement.plugin, { model: 'Incident', field: 'idnum', startAt: 1 });
 
 schema.listenTo(Report, 'change:incident', function(prevIncident, newIncident) {
-  Incident.findById(prevIncident || newIncident, function(err, incident) {
-    if (err || !incident) return;
-    var total = incident.totalReports;
-
-    if (prevIncident) {
-      total = (total > 0) ? total - 1 : 0;
-    } else if (newIncident) {
-      total = (total) ? total + 1 : 1;
-    }
-
-    incident.totalReports = total;
+  if (prevIncident !== newIncident) {
+    Incident.findByIdAndUpdate(prevIncident, { $inc: { totalReports: -1 } }, function(err, incident) {
+      if (err || !incident) return;
+    });
+  }
 
-    incident.save();
+  Incident.findByIdAndUpdate(newIncident, { $inc: { totalReports: 1 } }, function(err, incident) {
+    if (err || !incident) return;
```

Comment:
This callback does nothing. Better to use _.identity and a comment, or better yet, handle errors. Similarly above.